variables.tf

```
@@ -160,3 +160,18 @@ variable "ipv6_egress_only_internet_gateway_enabled" {
   description = "A boolean flag to enable/disable IPv6 Egress-Only Internet Gateway creation"
   default     = false
 }
+variable "adopt_default_route_table" {
+  type        = bool
+  description = "A boolean flag to enable/disable adoption of the default route table"
```

Change `A boolean flag` to `Whether` for each new input please

```suggestion
  description = "Whether to enable/disable adoption of the default route table"
```